src/FINDOLOGIC/Export/Helpers/DataHelper.php

```diff
@@ -43,6 +43,23 @@ public function __construct($propertyKey)
     }
 }
 
+class ValueExceedsInternalCharacterLimitException extends \RuntimeException
```

Comment:
The limit used by this exception specifically refers to attribute, so the exception name can be less generic and more concise, e.g. `AttributeValueLengthException`.

Reply:
Done :heavy_check_mark: